packages/smooth_app/lib/pages/preferences/user_preferences_widgets.dart

```diff
@@ -235,9 +235,7 @@ class UserPreferencesTitle extends StatelessWidget {
         ),
         child: Text(
           label,
-          style: Theme.of(context).textTheme.displaySmall?.copyWith(
-                height: 2.5,
-              ),
+          style: const TextStyle(fontSize: 24, fontWeight: FontWeight.bold),
```

Comment:
Could you keep the 

`Theme.of(context).textTheme.displaySmall?.copyWith(` 

and do your changed in this copy with. This allows us to use the global text style. 
Say we plan to change the text font we can just change it in the global theme and not at every text in the app

Reply:
Thankyou @M123-dev for guiding me. I will changes right away.

Reply:
@M123-dev I have made changes as you said.